lang/python/python3-bottle/Makefile

```diff
@@ -30,6 +30,7 @@ define Package/python3-bottle
 	TITLE:=Bottle is a fast, simple and lightweight WSGI micro web-framework for Python
 	URL:=https://bottlepy.org
 	DEPENDS:=+python3
+	PKGARCH:=all
```

Comment:
If this hasn't been updated to generate byte-code instead of sticking with source, it probably soon will be as that is how OpenWrt does Python..

Reply:
I only found this python package using source (.py) in ipk.
Technical details: found them by `grep -e 'define Build/Compile.endef'`

Reply:
What's your opinion? Should I drop this line now or only when it's updated to generate byte-code?

Reply:
I think @commodo should probably comment on that.  I'm of two minds on the answer, and he's the main Python on OpenWrt person.